src/components/GroupHeader.tsx

```diff
@@ -0,0 +1,39 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React from 'react';
+import { StyleProp, StyleSheet, TouchableOpacity, ViewStyle } from 'react-native';
+
+import { colors, consts, Icon, normalize, texts } from '../config';
+import { QUERY_TYPES } from '../queries';
+import { ScreenName } from '../types';
+
+const { a11yLabel, ROOT_ROUTE_NAMES } = consts;
+
+type Props = {
+  navigation: StackNavigationProp<any>;
+  style: StyleProp<ViewStyle>;
+};
+
+export const GroupHeader = ({ navigation, style }: Props) => (
+  <TouchableOpacity
+    onPress={() =>
+      navigation.navigate({
+        name: ScreenName.VolunteerForm,
+        params: {
+          title: texts.volunteer.groupNew,
+          query: QUERY_TYPES.VOLUNTEER.GROUP,
+          rootRouteName: ROOT_ROUTE_NAMES.VOLUNTEER
+        }
```

Comment:
could `useVolunteerNavigation` be used here?

Reply:
not could, it should!

Reply:
same for `CalendarHeader`

Reply:
21c78f0